tests/modules/ecs6-class/line.test.js

```diff
@@ -0,0 +1,65 @@
+const Line  = require('../../../modules/ecs6-class/line')
+const Point = require('../../../modules/ecs6-class/point')
+
+describe('CONSTRUCTOR',() => {
+    it('should build an object with default values',() => {
+        const line = new Line({slope:5,n:4})
+        expect(line.slope).toBe(5)
+        expect(line.n).toBe(4)
+        expect(line.point1.x).toBe(0)
+    })
```

Comment:
Where are all the tests for different cases: point1= [1,2,3], point1='string', n=new Point() and more